File: Develop/Backend/src/main/java/com/hientran/sohebox/sco/CryptoPortfolioHistorySCO.java
package com.hientran.sohebox.sco;

import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * @author hientran
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class CryptoPortfolioHistorySCO extends BaseSCO {

    private static final long serialVersionUID = -68140570073293062L;

    private SearchNumberVO id;

    private SearchDateVO timeStamp;

    private SearchNumberVO user;
}

File: Develop/Frontend/src/app/pages/crypto/_sco/cryptoPortfolioHistorySCO.ts
import { BaseSCO } from "../../../_common/_sco/baseSCO";
import { SearchNumber, SearchText } from "../../../_common/_sco/core_sco";

export class CryptoPortfolioHistorySCO extends BaseSCO {
  user: SearchNumber;
  token: SearchNumber;
}
